pinot-query-planner/src/test/resources/queries/PinotHintablePlans.json

```diff
@@ -1,6 +1,26 @@
 {
   "pinot_hint_option_tests": {
     "queries": [
+      {
+        "description": "hint table without partitioning should throw exception",
+        "sql": "EXPLAIN PLAN FOR SELECT * FROM d /*+ tableOptions(partition_key='col1', partition_size='4') */ LIMIT 10",
+        "expectedException": "Error composing query plan for:.*"
```

Comment:
TODO: fix this test, it should always check the nested reason b/c that's always wrapped in parser throw or planner throw. doesn't make sense to check just the top level msg